src/components/views/settings/tabs/user/PreferencesUserSettingsTab.js

```diff
@@ -122,6 +132,10 @@ export default class PreferencesUserSettingsTab extends React.Component {
         PlatformPeg.get().setMinimizeToTrayEnabled(checked).then(() => this.setState({minimizeToTray: checked}));
     };
 
+    _onTrayIconBlinkChange = (checked) => {
+        PlatformPeg.get().setTrayIconBlinkEnabled(checked).then(() => this.setState({trayIconBlink: checked}));
```

Comment:
By "device level" you mean stored in the account settings on the server? I'm not sure what the difference is and why the tray icon presence and its behavior should be treated differently.

Reply:
Tray icon presence needs to be stored in the desktop side so that it knows what to do before the webapp loads.

Reply:
I store and retrieve the blinking option using `platform.getTrayIconBlinkEnabled();` which then returns it from the desktop layer store. Can you point me to any examples of the desired behavior if this is incorrect?

Reply:
An example of this would be `minimizeToTray` - see definitions https://github.com/vector-im/element-desktop/blob/develop/src/electron-main.ts#L268-L274 and look in that file for other references

Reply:
Thanks, I'm not sure I'll be working on this any time soon, it's been long enough and people seem to have adapted to the current behavior. But if someone wants to continue let's keep this PR around.